firmware/controllers/algo/fuel_math.cpp

```diff
@@ -92,7 +92,7 @@ float getCrankingFuel3(
 	return crankingFuel;
 }
 
-floatms_t getRunningFuel(floatms_t baseFuel DECLARE_ENGINE_PARAMETER_SUFFIX) {
+float getRunningFuel(float baseFuel DECLARE_ENGINE_PARAMETER_SUFFIX) {
```

Comment:
let's avoid unrelated changes in a PR?

many small PRs is greatly preferred over one big PR

Reply:
Yeah I can rip that stuff out and submit it separately.